### PR TITLE
[untag] Add untag_entry and untag_entries method

### PR DESCRIPTION
**Pull request by Mathieu4141**

### CODE REVIEW

File: feedly/api_client/data.py [hunk 1]
```diff
@@ -76,6 +76,17 @@ def tag_entries(self, entry_ids: List[str]):
...
+
+    def untag_entries(self, entry_ids: List[str]):
+        # limitation due to the url length: articles are "de-tagged" by batch of 50.
+        for i in range(0, len(entry_ids), 50):
+            self._client.do_api_request(
+                f'/v3/tags/{quote_plus(self["id"])}/{",".join([quote_plus(d) for d in entry_ids[i: i+50]])}',
+                method='DELETE',
+            )
+
     def delete_tags(self, options: StreamOptions = None):
```
Comment (Edouard360) [inline feedly/api_client/data.py]: delete_all_tags, delete_tag_for_entry, ,delete_tag_for_entries?

File: feedly/api_client/data.py [hunk 2]
```diff
@@ -76,6 +76,17 @@ def tag_entries(self, entry_ids: List[str]):
...
                                     data={'entryIds': [entry_id for entry_id in entry_ids]})
 
+    def untag_entry(self, entry_id: str):
+        self.untag_entries([entry_id])
+
+    def untag_entries(self, entry_ids: List[str]):
+        # limitation due to the url length: articles are "de-tagged" by batch of 50.
+        for i in range(0, len(entry_ids), 50):
+            self._client.do_api_request(
+                f'/v3/tags/{quote_plus(self["id"])}/{",".join([quote_plus(d) for d in entry_ids[i: i+50]])}',
```
Comment (kireet) [inline feedly/api_client/data.py]: i think you can do 
POST /v3/entries/.mdelete 
with the content being the list of entry ids.

Comment (Mathieu4141) [reply feedly/api_client/data.py]: This is not on the API ? https://developer.feedly.com/v3/tags/#delete-tags
How do you precise which tag to delete then ?

Comment (kireet) [reply feedly/api_client/data.py]: you're right, .mdelete is not available on this api endpoint

File: feedly/api_client/data.py [hunk 3]
```diff
@@ -76,7 +78,18 @@ def tag_entries(self, entry_ids: List[str]):
         self._client.do_api_request(f'/v3/tags/{quote_plus(self["id"])}', method='put',
                                     data={'entryIds': [entry_id for entry_id in entry_ids]})
 
-    def delete_tags(self, options: StreamOptions = None):
+    def delete_tag_for_entry(self, entry_id: str):
```
Comment (kireet) [inline feedly/api_client/data.py]: sorry i didn't comment before. i actually think untag_entry/entries sounds more natural. delete_tag_for_entry sounds a bit weird.

**Pull request merged**